docs/site-example/i18n/en.po

```diff
@@ -30,4 +30,6 @@ msgstr ""
 "nearby Freifunk nodes  after that. "
 "Your can find lots of information on the Freifunk Duckburg community on "
 "<a href=\"https://entenhausen.freifunk.net/\">our homepage</a>.</p>"
+"<p>In order to re-enter Configmode, press the reset button during normal "
+"operation for 3 seconds. The device will reboot in Configmode.</p>"
```

Comment:
Proposal: "To get back to this configuration interface, press the reset button for 3 seconds during normal operation. The device will then reboot into config mode."